### PR TITLE
Login-Modal

### PR DESCRIPTION
**Pull request by Aleena-4**
alreadyreg
forgotp
login

### CODE REVIEW

File: src/libs/ui/src/atoms/Modal/modal.styles.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  display: none;
+}
+/* .modal-content {
+  background-color: var(--brand-white, #ffffff); 
+  border-radius: 0.5rem; 
+  overflow-y: auto;
+  padding: 1rem;
+  max-width: 90vw;
+  max-height: 90vh; 
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1); /
```
Comment (Vishnu9901) [inline src/libs/ui/src/atoms/Modal/modal.styles.css:44]: please remove commented code

File: src/libs/ui/src/atoms/PasswordField/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export const PasswordFeild: React.FC<FormFieldProps> = ({ type, placeholder, className, suffix, ...props }) => {
+  return (
+    <div className="relative w-full">
+      <input
+        type={type}
+        placeholder={placeholder}
+        className={`${className} w-full`}
+        {...props}
+      />
+      {suffix && (
```
Comment (Vishnu9901) [inline src/libs/ui/src/atoms/PasswordField/index.tsx:20]: please give proper names to variables

File: src/libs/ui/src/atoms/RegisterModal/registerBody.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface registerBodyProps {
+    children: React.ReactNode;
+}
+
+const registerBody: React.FC<registerBodyProps> = ({ children }) => {
```
Comment (Vishnu9901) [inline src/libs/ui/src/atoms/RegisterModal/registerBody.tsx:7]: Please remove unused code

File: src/libs/ui/src/atoms/RegisterModal/registerHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface registerHeaderProps {
+    heading : string
+}
+
+const registerHeader: React.FC<registerHeaderProps> = ({ heading }) => {
```
Comment (Vishnu9901) [inline src/libs/ui/src/atoms/RegisterModal/registerHeader.tsx:7]: please remove unused code

File: src/libs/ui/src/molecules/AlreadyRegisteredForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                      className="text-gray-600 focus:outline-none"
+                    >
+                      {showPassword ? 'Hide' : 'Show'}
+                    </button>
+                  )
+                )}
+              />
+            }
+            control={control}
+            {...register(LoginFormEnum.Password, { required: 'This field is required' })}
```
Comment (Vishnu9901) [inline src/libs/ui/src/molecules/AlreadyRegisteredForm/index.tsx:85]: move these messages to constants file

File: src/libs/ui/src/molecules/ForgotPasswordModal/index.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          control={control}
+          {...register('email', { required: 'Email is required' })}
+        />
+        {errors.email && <span style={{ color: '#ce4635' }}>{errors.email.message}</span>}
+
+        <div className="flex justify-end items-center">
+        <Button 
+  onClick={() => setIsForgotPassword(false)} // Go to Already Registered form
+  className="bg-none text-blue-600" 
+  type={undefined}
```
Comment (Vishnu9901) [inline src/libs/ui/src/molecules/ForgotPasswordModal/index.tsx:51]: pass proper button type

File: src/libs/ui/src/molecules/LoginForm/index.tsx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+          {errors.password && (
+            <span style={{ color: '#ce4635' }} className="text-normal font-bold mt-3">
+              {errors[LoginFormEnum.Password]?.message}
+            </span>
+          )}
+        </div>
+
+        <Button
+          onClick={() => setIsForgotPassword(true)}
+          type={undefined}
```
Comment (Vishnu9901) [inline src/libs/ui/src/molecules/LoginForm/index.tsx:110]: every file should be formatted

File: tailwind.config.js [hunk 8]
```diff
@@ -9,6 +9,16 @@ module.exports = {
         hero: ['Hero', 'sans-serif'],
         heroNewLight: ['Hero New Light', 'sans-serif'], // Add the custom font
       },
+      width: {
+        '40.6rem': '40.6rem', // Custom width value
+      },
+      height:{
+        '31rem': '31rem',
```
Comment (Vishnu9901) [inline tailwind.config.js:16]: please remove these changes

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

**Comment by Aleena-4** [comment]
decline this PR due to lot of merge conflicts

**Pull request closed without merging**